### ISSUE TITLE
Better proofreading in internal cross references 

### ISSUE THREAD

**Issue by simonnagl** [asciidoctor/asciidoctor-intellij-plugin#539]
Asciidoc supports internal cross references like this:

```
[[_l1]]
== Section to link to

Text between

Text with <<_l1,link>> and something
```

With this version, Intellij maks `<<_l1,link>> with a warning `repetition of spaces`.

The proofreading of Intellij should treat `Text with link` as text and exclude `<<_l1,>>` as syntax.

I do not know if this is in the scope of this plugin.
I do not know if this is possible for the plugin.

 

AsciiDoc Plugin Version: 0.31.21

IDE Name and Version:
IntelliJ IDEA 2020.2 (Ultimate Edition)
Build #IU-202.6397.94, built on July 27, 2020

**Comment by ahus1** [comment]
Thanks for reporting this. It is within the scope of the plugin. I'll have a look.

**Comment by ahus1** [comment]
I've published pre-release 0.31.22 that fixes this issue. Please give it a try. 

As the example you provided worked for me, I close this issue. If you find that it doesn't yet work for you, please comment and I'll re-open the issue. 

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
